src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.github.rodionovsasha.commentservice.services.impl;
+
+import com.github.rodionovsasha.commentservice.entities.User;
+import com.github.rodionovsasha.commentservice.exceptions.InactiveUserException;
+import com.github.rodionovsasha.commentservice.exceptions.UserNotFoundException;
+import com.github.rodionovsasha.commentservice.repositories.UserRepository;
+import com.github.rodionovsasha.commentservice.services.UserService;
+import lombok.AllArgsConstructor;
+import lombok.val;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@AllArgsConstructor
+@Service @Transactional
+public class UserServiceImpl implements UserService {
+    private final UserRepository repository;
+
+    @Override
+    public User create(String name, int age) {
+        return repository.saveAndFlush(new User(name, age));
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public User getById(long id) {
+        val user = repository.findOne(id);
+        if (user == null) {
+            throw new UserNotFoundException(String.format("The user with id '%d' could not be found", id));
```

Comment:
I think `UserNotFoundException` should consume only `id` and take care of the message itself. It is possible to do within public static methods right in the `UserNotFoundException`:
```java
public static void forUser(long id) {
  return new UserNotFoundException(String.format("The user with id '%d' could not be found", id));
}
```

The same should be applied for other exceptions